pandas/plotting/_core.py

```diff
@@ -1871,14 +1880,8 @@ def plot_series(data, kind='line', ax=None,                    # Series unique
                 **kwds):
 
     import matplotlib.pyplot as plt
-    """
-    If no axes is specified, check whether there are existing figures
-    If there is no existing figures, _gca() will
-    create a figure with the default figsize, causing the figsize=parameter to
-    be ignored.
-    """
     if ax is None and len(plt.get_fignums()) > 0:
-        ax = _gca()
+        ax = _gca(figsize)
```

Comment:
I would maybe revert this change, since we pass figsize through to `_plot`, presumably it's handled there.